<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReportsDialog</class>
 <widget class="QDialog" name="ReportsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1006</width>
    <height>824</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Reports</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,0,0,0">
   <item row="1" column="2">
    <widget class="QPushButton" name="selectSessionButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Select sessions</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="sessionsLabel">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Sessions</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0" colspan="5">
    <widget class="QTableWidget" name="dataTable">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="reportTypeLabel">
     <property name="text">
      <string>Report type</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="5">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="exportButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Export to CSV file</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="selectedSessionLabel">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>0 selected</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="4">
    <widget class="QComboBox" name="reportTypeComboBox">
     <property name="placeholderText">
      <string>Select a report type</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
